content/ngf/install/manifests/plus.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Use Manifests to install NGINX Gateway Fabric with NGINX Plus
+linkTitle: NGINX Plus
+weight: 200
+toc: true
+nd-content-type: how-to
+nd-product: FABRIC
+---
+
+This page describes how to use Manifests to install NGINX Gateway Fabric with NGINX Plus.
+
+It explains the requirements for NGINX Gateway Fabric, how to deploy NGINX Gateway Fabric, its custom resource definitions (CRDs) and the Gateway API resources.
+
+By following these instructions, you will finish with a functional NGINX Gateway Fabric instance for your Kubernetes cluster.
+
+## Before you begin
+
+To complete this guide, you will need the following pre-requisites:
+
+- An active NGINX Plus subscription (Purchased or trial)
+- [A supported Kubernetes version]({{< ref "/ngf/overview/technical-specifications.md" >}})
+- A functional Kubernetes cluster
+
```

Comment:
The cert manager stuff around secure certs is still valid for nginx plus.

Reply:
This is an oversight, as the instructions still explain using them. Updating this now.